python/cuml/cuml/cluster/kmeans_utils.pxd

```diff
@@ -24,14 +24,25 @@ cdef extern from "cuml/cluster/kmeans.hpp" namespace \
         "cuvs::cluster::kmeans::params":
     enum InitMethod:
         KMeansPlusPlus, Random, Array
+cdef extern from "raft/core/logger.hpp" namespace "raft":
```

Comment:
Maybe add a blank line between files.
```suggestion

cdef extern from "raft/core/logger.hpp" namespace "raft":
```

Reply:
Ack I had fixed that in a previous commit that I reverted. Thanks.